extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchBufferAggregator.java

```diff
@@ -113,4 +110,28 @@ public void close()
     unions.clear();
   }
 
+  @Override
+  public void relocate(int oldPosition, int newPosition, ByteBuffer newBuffer)
```

Comment:
I disagree with `relocate(int[] newPositions, ByteBuffer newBuffer)` interface b/c we still need old positions to dereference old `Union` built on top of old `ByteBuffer` stored in `unions` map